Library/Formula/scirius.rb

```diff
@@ -0,0 +1,126 @@
+class Scirius < Formula
+  desc "Rules manager for Suricata IDPS"
+  homepage "https://github.com/StamusNetworks/scirius"
+  url "https://github.com/StamusNetworks/scirius/archive/scirius-1.0.tar.gz"
+  sha256 "fde16b3ac8f95e2990a44a9d85aa7c3ee6b55e59af9c27ded7319db19c19b2f5"
+
+  depends_on "python"
```

Comment:
but in this case `pip` will not be installed and all the modules will fail, won't they?